### DISCUSSION TITLE
Tempdb mdf files not clearing

### DISCUSSION THREAD

**Discussion by thiggins1990** [olahallengren/sql-server-maintenance-solution#827] [General]
I switched from using the native sql backup function (the maintenance plan) to the Ola Hallengren script and I realized my Tempdb mdf files were growing out of control. I checked the tempdb and can a report showing top tables by disk usage and the tables had like 128mb of data, yet, there are 4 tempdb files of 28gb a piece. My only solution was to restart the sql service. Does the ola script empty the tempdb file? If not, can you add it as a feature?

**Comment by olahallengren** [comment]
The Maintenance Solution is not doing anything with respect to tempdb.

You need to find out which job/command that was causing tempdb to grow.
